packages/schematics/angular/application/schema.d.ts

```diff
@@ -25,11 +25,11 @@ export interface Schema {
     /**
      * Specifies the view encapsulation strategy.
      */
-    viewEncapsulation?: ('Emulated' | 'Native' | 'None');
+    viewEncapsulation?: ("Emulated" | "Native" | "None");
```

Comment:
Unfortunately you have to do a `devkit-admin lint --fix` run after generating the `.d.ts`. This won't pass linting.